### PR TITLE
Added Account Enrichment v2.1 + Test v2.1

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
In Progress - need documentation

## Related Issues
Related to: https://github.com/demisto/etc/issues/16881

## Description
Added the `Account Enrichment - Generic v2.1` playbook. The playbook features the following improvements over the previous one:
- Changed AD to AD v2
- Updated descriptions of task names / playbook
- Removed redundant outputs

Also added the test playbook `Account Enrichment - Generic v2.1 - Test`.

## Screenshots
Playbook
Account_Enrichment_-_Generic_v2 1_Thu_May_02_2019

Test playbook
Account_Enrichment_-_Generic_v2 1_-_Test_Thu_May_02_2019

## Required version of Demisto
4.1.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation
- [ ] Code Review

### CODE REVIEW

File: Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    ignoreworker: false
+  "5":
+    id: "5"
+    taskid: 97ec5ca3-8754-49ff-8797-627ef7e6cd33
+    type: regular
+    task:
+      id: 97ec5ca3-8754-49ff-8797-627ef7e6cd33
+      version: -1
+      name: Get account info from Active Directory
+      description: Queries Active Directory for the username’s information.
```
Comment (kirbles19) [inline Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml]: Queries Active Directory and returns information for the specified username.

File: Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      transformers:
+      - operator: uniq
+  required: false
+  description: The username to enrich.
+outputs:
+- contextPath: Account
+  description: The account object.
+  type: unknown
+- contextPath: ActiveDirectory.Users.sAMAccountName
+  description: The user's sAMAccountName.
```
Comment (kirbles19) [inline Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml]: Is this the actual name of it: sAMAccountName ?

File: Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+- contextPath: ActiveDirectory.Users.userAccountControl
+  description: The user's account control flag.
+- contextPath: ActiveDirectory.Users.mail
+  description: The user's email address.
+- contextPath: ActiveDirectory.Users.memberOf
+  description: Groups the user is a member of.
+releaseNotes: "Replaced old Active Directory integration with the new Active Directory v2 Query integration.
```
Comment (kirbles19) [inline Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml]: Replaced the Active Directory integration with the Active Directory v2 Query integration.

File: Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml (hunk 2 above)
Comment (idovandijk) [reply Playbooks/playbook-Account_Enrichment_-_Generic_v2.1.yml]: In Demisto they called it "sAMAccountName". The actual term is "samAccountName". Will change the description to samAccountName.

**Pull request merged**